.github/workflows/testpypi-publish-kensu-pyspark-on-pr.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.7 ]
```

Comment:
this special `kensu-pyspark` build was made explicitly for OLD python version (extracting only subset of `kensu-py` files which can work with old python), as someone needed support for out-dated `py3.7` . 
@SammyEK maybe we actually don't need it anymore at all?